### ISSUE TITLE
feature: better share urls

### ISSUE THREAD

**Issue by Entkenntnis** [Entkenntnis/robot-karol-online#279]
I envision something like https://karol.arrrg.de/X73C

**Comment by Entkenntnis** [comment]
will need a `.htaccess` file like this:

```
RewriteEngine On
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(.*)$ /
```

(I don't want to provide a custom server - just plain html)

**Comment by Entkenntnis** [comment]
enable this feature only in production. Local deployment should still use a safer `/?id=xxx` approach

(rule here maybe: no documentation of backend (yet), but everything should work locally)

**Comment by Entkenntnis** [comment]
compromise: https://karol.arrrg.de/#x73c
-> no need for server side configuration, works on localhost

**Comment by Entkenntnis** [comment]
aim for shorter and less confusing links that are easy to type

**Comment by Entkenntnis** [comment]
postponed after #231
